src/features/cartographie/components/lieux-mediation-numerique-details/accessibilite/accessibilite.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="bg-light p-2 d-print-none d-flex">
+  <div class="d-flex flex-grow-1 mt-3">
```

Comment:
```suggestion
  <div class="d-flex flex-grow-1">
```
L'espace en haut doit être de la même taille que l'espace en bas